scene/2d/visibility_notifier_2d.cpp

```diff
@@ -127,235 +108,106 @@ void VisibilityNotifier2D::_bind_methods() {
 
 	ADD_PROPERTY(PropertyInfo(Variant::RECT2, "rect"), "set_rect", "get_rect");
 
-	ADD_SIGNAL(MethodInfo("viewport_entered", PropertyInfo(Variant::OBJECT, "viewport", PROPERTY_HINT_RESOURCE_TYPE, "Viewport")));
```

Comment:
These are gone, so its not possibleany longer to tell visibility per viewport. I dont know if anyone used this, but you will have to find a different workaround if this is the case. Its the cost to pay for a much simpler and more efficient system.